lighthouse-core/test/audits/user-timing-test.js

```diff
@@ -22,11 +22,6 @@ describe('Performance: user-timings audit', () => {
   it('evaluates valid input correctly', () => {
     const artifacts = generateArtifactsWithTrace(traceEvents);
     return UserTimingsAudit.audit(artifacts, {computedCache: new Map()}).then(auditResult => {
-      const blackListedUTs = auditResult.extendedInfo.value.filter(timing => {
-        return UserTimingsAudit.blacklistedPrefixes.some(prefix => timing.name.startsWith(prefix));
-      });
-      assert.equal(blackListedUTs.length, 0, 'Blacklisted usertimings included in results');
```

Comment:
this is probably worth preserving too. It's implicitly checked in the assertions below since the `goog_*` events don't appear there (and presumably they're in the trace), but it's nice to call it out and have a clear error -> what to fix in case that regresses somehow.